.github/workflows/build.yml

```
@@ -0,0 +1,289 @@
+      matrix:
+        platform: [linux/amd64, linux/arm64]
+        include:
+          - platform: linux/amd64
+            platform_slug: amd64
+          - platform: linux/arm64
+            platform_slug: arm64
+    outputs:
+      activitypub-private-tags: ${{ steps.activitypub-docker-metadata-private.outputs.tags }}
+      activitypub-public-tags: ${{ steps.activitypub-docker-metadata-public.outputs.tags || '' }}
+      activitypub-public-labels: ${{ steps.activitypub-docker-metadata-public.outputs.labels || '' }}
+      activitypub-migrations-private-tags: ${{ steps.activitypub-migrations-docker-metadata.outputs.tags }}
```

actionlint (1.7.7)

property "activitypub-migrations-docker-metadata" is not defined in object type {activitypub-docker-metadata-private: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-docker-metadata-public: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-migrations-docker-metadata-private: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}; activitypub-migrations-docker-metadata-public: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}}

(expression)